src/index.html

```diff
@@ -11,7 +11,108 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post" onsubmit="onSubmit()">
+      <fieldset class="form_fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form__input">
+          <label for="surname">Surname:</label>
+          <input type="text" id="surname"
+            name="surname" autocomplete="off">
```

Comment:
If there are more then 2 attributes on tag, each should be on the new line, fix it for all of your tags
```suggestion
          <input 
          	type="text" 
          	id="surname"
            name="surname" 
            autocomplete="off"
          >
```